src/Symfony/Component/Validator/composer.json

```diff
@@ -21,7 +21,7 @@
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/polyfill-php80": "^1.15",
         "symfony/translation-contracts": "^1.1|^2",
-        "symfony/deprecation-contracts": "^2.0"
+        "symfony/deprecation-contracts": "^2.1"
```

Comment:
The first `symfony/deprecation-contracts` was 2.1.0, so your version constraint is a bit more accurate, but I don't expect your change to have any effect.

Reply:
Apparently yes, but other components have version 2.1 :)

Reply:
If it doesn't add anything to solve #38000, I would revert this change.

Reply:
@xabbuh Done.